docs/guides/admin/docs/installation/source-linux.md

```diff
@@ -54,6 +54,7 @@ Required:
 Required (not necessarily on the same machine):
 
     ActiveMQ >= 5.10 (older versions untested)
+    Elasticsearch >= 7.9.2 (see the search module's pom file for the correct vertion)
```

Comment:
While `9.0 ≥ 7.9.2` Elasticsearch 9 would not work. Also, I don't think that referencing a modules `pom.xml` file is very helpful for new users

```suggestion
    Elasticsearch 7.9.x
```